app/controllers/api/verify/password_reset_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Api
+  module Verify
+    class PasswordResetController < Api::BaseController
+      def create
+        analytics.idv_forgot_password_confirmed
+        request_id = sp_session[:request_id]
+        email = current_user.email
+        reset_password(email, request_id)
+
+        render json: { redirect_url: forgot_password_url(request_id: request_id) },
+               status: :accepted
+      end
+
+      private
+
+      def reset_password(email, request_id)
+        sign_out
```

Comment:
this immediately wipes out the session (which matches existing behavior) but now that we have more of a single page app... do we need to signal that to the UI at all? Or I guess passing and going to the `redirect_url` handles that?

Reply:
> this immediately wipes out the session (which matches existing behavior) but now that we have more of a single page app... do we need to signal that to the UI at all? Or I guess passing and going to the `redirect_url` handles that?

Yeah, I'm still finding my footing as far as how much awareness the server should have of a user session, but in this case, it seems to make sense that the client doesn't need to be too aware of what's happening on the server beyond expecting to need to redirect somewhere after the response?